### DISCUSSION TITLE
Log4j Vulnerability with Atom Text Editor

### DISCUSSION THREAD

**Discussion by bacmic** [atom/atom#23404] [Q&A]
Hello,

Our vulnerability mgmt team flagged a critical vulnerability in Log4J.

Is your technology affected by the Log4J vulnerability (also known as CVE-2021-44228 and CVE-2021-45046)?

**Comment by icecream17** [comment]
No one really knows, doesn't seem like it, but the main discussion is here: https://github.com/atom/atom/discussions/23378
